OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory70.cs

```diff
@@ -50,66 +50,64 @@ protected override unsafe Combatant GetCombatantFromByteArray(byte[] source, uin
                     OwnerID = mem.OwnerID == emptyID ? 0 : mem.OwnerID,
                     Type = (ObjectType)mem.Type,
-                    MonsterType = 0,
                     Status = (ObjectStatus)mem.Status,
-                    ModelStatus = (ModelStatus)mem.ModelStatus,
-                    // Normalize all possible aggression statuses into the basic 4 ones.
-                    AggressionStatus = 0,
-                    NPCTargetID = mem.NPCTargetID,
                     RawEffectiveDistance = mem.EffectiveDistance,
                     PosX = mem.PosX,
                     // Y and Z are deliberately swapped to match FFXIV_ACT_Plugin's data model
                     PosY = mem.PosZ,
                     PosZ = mem.PosY,
                     Heading = mem.Heading,
                     Radius = mem.Radius,
-                    // In-memory there are separate values for PC's current target and NPC's current target
-                    TargetID = (ObjectType)mem.Type == ObjectType.PC ? mem.PCTargetID : mem.NPCTargetID,
-                    CurrentHP = mem.CurrentHP,
-                    MaxHP = mem.MaxHP,
-                    Effects = exceptEffects ? new List<EffectEntry>() : GetEffectEntries(mem.Effects, (ObjectType)mem.Type, mycharID),
-
                     BNpcID = mem.BNpcID,
-                    CurrentMP = mem.CurrentMP,
-                    MaxMP = mem.MaxMP,
-                    CurrentGP = mem.CurrentGP,
-                    MaxGP = mem.MaxGP,
-                    CurrentCP = mem.CurrentCP,
-                    MaxCP = mem.MaxCP,
-                    Level = mem.Level,
-                    PCTargetID = mem.PCTargetID,
-
-                    BNpcNameID = mem.BNpcNameID,
-
-                    WorldID = mem.WorldID,
-                    CurrentWorldID = mem.CurrentWorldID,
-
-                    IsCasting1 = mem.IsCasting1,
-                    IsCasting2 = mem.IsCasting2,
-                    CastBuffID = mem.CastBuffID,
-                    CastTargetID = mem.CastTargetID,
-                    // Y and Z are deliberately swapped to match FFXIV_ACT_Plugin's data model
-                    CastGroundTargetX = mem.CastGroundTargetX,
-                    CastGroundTargetY = mem.CastGroundTargetZ,
-                    CastGroundTargetZ = mem.CastGroundTargetY,
-                    CastDurationCurrent = mem.CastDurationCurrent,
-                    CastDurationMax = mem.CastDurationMax,
-
-                    TransformationId = mem.TransformationId,
-                    WeaponId = mem.WeaponId
+                    Address = (IntPtr)p,
                 };
-                combatant.IsTargetable =
-                    (combatant.ModelStatus == ModelStatus.Visible)
-                    && ((combatant.Status == ObjectStatus.NormalActorStatus) || (combatant.Status == ObjectStatus.NormalSubActorStatus));
-                if (combatant.Type != ObjectType.PC && combatant.Type != ObjectType.Monster)
+                
```

Comment:
Maybe add a comment here reiterating what is stated below, something like `These properties only exist on larger combatants`, or something along those lines?

Additionally, this logic change should be reflected in `CombatantMemory65.cs` as well, for consistency.